query/security/wcpm/src/main/java/org/codice/ddf/admin/security/wcpm/actions/persist/SaveContextPolices.java

```diff
@@ -60,29 +71,53 @@ public SaveContextPolices(ConfiguratorFactory configuratorFactory) {
 
     @Override
     public ListField<ContextPolicyBin> performAction() {
+        Configurator configurator = configuratorFactory.getConfigurator();
+        configurator.updateConfigFile(POLICY_MANAGER_PID,
+                new PolicyManagerServiceProperties().contextPoliciesToPolicyManagerProps(contextPolicies.getList()),
+                true);
+
+        // TODO: 4/14/17 Fix the contextPolicies.toString(), this will print the objects name
```

Comment:
I hate having to provide a messages to the configurator, it should be documenting the changing automatically. We're going to continually forget to add these to the logging, I think it should be automated.